com.unity.render-pipelines.universal/Documentation~/urp-renderer-feature.md

```diff
@@ -0,0 +1,28 @@
+# URP Renderer Feature
+
+A Renderer Feature is an asset that lets you add extra Render passes to a URP Renderer and configure their behavior.
+
+URP contains the pre-built Renderer Feature called **Render Objects**.
+
+For information on how to add a Renderer Feature to a Renderer, see the page [How to add a Renderer Feature to a Renderer](urp-renderer-feature-how-to-add.md).
+
+## Render Objects Renderer Feature<a name="render-objects-renderer-feature"></a>
+
+The Render Objects Renderer Feature contains the following properties.
+
+![Render Objects Renderer Feature Inspector view](Images/urp-assets/urp-renderer-feature-render-objects.png)
+
+| Property | Description |
+|:-|:-|
+| **Name** | The name of the feature. |
+| **Event** | The event in the URP queue when Unity executes this Renderer Feature. |
```

Comment:
This seems grammatically incorrect.

Do we talk about things that URP does as URP or Unity? Because Renderer Features are a feature completely unknown to core Unity, and is handled by URP itself.